sys/include/net/gcoap.h

```diff
@@ -504,11 +504,14 @@ void gcoap_register_listener(gcoap_listener_t *listener);
  * @param[in] code      Request code: GCOAP_[GET|POST|PUT|DELETE]
  * @param[in] path      Resource path, *must* start with '/'
  *
+ * @pre `path != NULL`
+ * @pre `path[0] == '/'`
```

Comment:
I would write something like that verbatim, like "Path *must* start with a `/`"

Reply:
agreed, but I often see this notation in our doc, too. Anyway, then the comment can be removed from the param above, right?